### ISSUE TITLE
<CAIA Intake> from <Chatbot Benefits Micro-Services>

### ISSUE THREAD

**Issue by leahspix** [department-of-veterans-affairs/va.gov-team#78614]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: `Chatbot Benefits-Microservices`
- **OCTO product owner:** `Luciana Morais`
- **Product name:** `Chatbot`
- **Product manager:**  `Leah Spix`
- **Slack channel:** `#va-chatbot-claims`
- **Dedicated content writer on your team (if you have one):** `Grace Goodenough, goodenoughgraceVA`
- **Dedicated a11y specialist on your team (if you have one):** `Insert name and/or GH handle`
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

`Tell us briefly about your work and the outcomes you’re aiming to achieve.`
Design and build an updated Disability Claims Status Flow as part of the VA.gov Chatbot. The new Claims Flow will include: Bot can provide a) a definition of the phase, b) a general claim timeline, and/or c) details within the phase (micro-transaction). The bot can direct users to other VA tools (ex. Claims Status Tool, VA.gov pages) where information can be found.
- The Chatbot will share the number of compensation claims on file, the most recent claim opened, the date it was opened, the related issues, and the status
- The Chatbot will share details on the step of the claim process that disability compensation claim is in including the current status of the claim: claim received, initial review, evidence gathering review decision, preparation for notification, complete
- The Chatbot will share when the claim moved status/ the date the status changed
- The Chatbot will provide recent updates for the veteran to learn about the last activity done to the compensation claim. The chatbot will share the activity, date submitted, and any actions needed from the veteran
- The Chatbot will provide the veteran with the average number of days it takes to complete a disability compensation claim (provided by VA.gov) as of the most relevant date
- The Chatbot will provide the veteran with a phone number (1-800-827-1000) to click-to-call to answer questions related to disability claims status that the chatbot cannot answer

At this time, this chatbot can only show a Veteran status information on disability compensation claims. The chatbot will provide the date the claim was filed and the current phase that the compensation claim is in. All other details of the in-progress claims and other claims must be found via the Claims Status Tool or through an agent. To provide a seamless experience across VA platforms and channels, the Chatbot should have the same capabilities in this area as other tools on the VA platform, such as the Claims Status Tool. Responding to select claims status details and not providing details to the veteran could harm the veteran's confidence in the bot over the long term.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [ ] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.
- In Design Intent Phase of the Collaboration Cycle 
- Milestones are in the [Chatbot Claims Mural board](https://app.mural.co/t/bahdigitalexperience6902/m/bahdigitalexperience6902/1701799055287/d5c1955ce52357dc4ab722475e0e4b897d744a97?wid=0-1704204590877&outline=open)
- Release Date: April 11, 2024

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `78076`

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Product Initiative Brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/virtual-agent/product/claims-status-initiative-brief.md) 

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA operations
- [ ] Initial Review - @coforma-terry and @strelichl 
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
```

```[tasklist]
### CAIA IA tasks
```

**Comment by sara-amanda** [comment]
[Per SK via Slack](https://oddball.slack.com/archives/C05LB36ATAQ/p1727986457072969?thread_ts=1727701900.927949&cid=C05LB36ATAQ): 
> Anita said to close 78614 and 73543
